protobuf/src/main/java/com/linecorp/armeria/server/protobuf/ProtobufResponseConverterFunction.java

```diff
@@ -142,6 +145,30 @@ public ProtobufResponseConverterFunction(Printer jsonPrinter) {
         this.jsonPrinter = requireNonNull(jsonPrinter, "jsonPrinter");
     }
 
+    @Override
+    public Boolean isResponseStreaming(Type returnType, @Nullable MediaType produceType) {
+        final Class<?> clazz = typeToClass(unwrapAsyncType(returnType));
+        if (clazz == null) {
+            return null;
+        }
+
+        // Non-streaming types
+        if (Message.class.isAssignableFrom(clazz)) {
+            return false;
```

Comment:
I found out that there's case we fall through when the clazz is a Message. Could you check it, please?

https://github.com/line/armeria/pull/4177/files#diff-fd2e367dd8f05b6d82bf8ef3e10ece9a71205cb5e0229015c2408d8cc84e18c0R222